derive-impl/src/multihash.rs

```diff
@@ -247,125 +247,14 @@ pub fn multihash(s: Structure) -> TokenStream {
         }
 
         impl core::convert::TryFrom<u64> for #code_enum {
-            type Error = #mh_crate::Error;
+            type Error = #mh_crate::UnsupportedCode;
 
             fn try_from(code: u64) -> Result<Self, Self::Error> {
                 match code {
                     #(#code_from_u64,)*
-                    _ => Err(#mh_crate::Error::UnsupportedCode(code))
+                    _ => Err(#mh_crate::UnsupportedCode(code))
                 }
             }
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn test_multihash_derive() {
```

Comment:
> Is there a way to revive this test? I think it's valuable that the generated code doesn't change.

I tried to port it but I couldn't make it work with the new dependency layout, hence I nuked it and replaced with `trybuild` tests. On the whole, I think it is better to test the features you want from your custom-derive rather than the generated code iteslf. The new tests will make sure that stuff compiles f.e.

Reply:
It's helpful when developing the proc macro or when there was an issue with proc_macro_crate. But I guess there won't be many changes in the future anyway.